tests/e2e/cypress/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: [
+    '../node_modules/@centreon/frontend-core/eslint/react/typescript.eslintrc.js',
+    'plugin:cypress/recommended',
```

Comment:
Once Cypress will be integrated in other modules, we might have to move this line on the frontend-core project

Reply:
yes indeed